frontend/packages/kubevirt-plugin/src/components/modals/cd-rom-modal/index.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { getNamespace } from '@console/shared';
+import { Firehose } from '@console/internal/components/utils';
+import { createModalLauncher, ModalComponentProps } from '@console/internal/components/factory';
+import { PersistentVolumeClaimModel, ProjectModel, NamespaceModel } from '@console/internal/models';
+import { getResource } from 'kubevirt-web-ui-components';
+import { VMKind } from '../../../types';
+import { VirtualMachineModel } from '../../../models';
+import { VMCdRomModal } from './cd-rom-modal';
+
+const CDVModalFirehose: React.FC<CloneVMModalFirehoseProps> = (props) => {
```

Comment:
why `CDV` ?

Reply:
suppose to be CDVMModal..
do you have a better name in mind?

Reply:
I'd go with `CDRomModal`